### ISSUE TITLE
哔哩哔哩视频投稿up主名称为undefined

### ISSUE THREAD

**Issue by zsbai** [DIYgod/RSSHub#11050]
### 路由地址

```routes
/bilibili/user/channel/:uid/:sid/:disableEmbed?
```

### 完整路由地址

```fullroutes
bilibili/user/channel/2267573/396050
```

### 相关文档

https://docs.rsshub.app/social-media.html#bilibili-up-zhu-pin-dao-de-shi-pin-lie-biao

### 预期是什么？

《up主名称》 的 bilibili 频道

### 实际发生了什么？

undefined 的 bilibili 频道

### 部署

RSSHub 演示 (https://rsshub.app)

### 额外信息

```shell
自建和官方均显示undefined，使用一些rss阅读器会把所有订阅b站的标题都改成 undefined的视频/频道
```

### 这不是重复的 issue

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保该错误尚未被报告。

**Comment by TonyRL** [comment]
/test
```routes
/bilibili/user/channel/2267573/396050
```

**Comment by TonyRL** [comment]
> The undefined title is still returned in my vercel service.

Please make sure your self-hosted instance is updated. The above-mentioned issue is fixed in 6fdd85e0fed0afa1cf3af25f0966e0904216235c

> it is still present in the latest official service

> demo instance is shared which means some websites, especially those sites having a `strict anti-crawler policy` badge, may **impose** **rate limits** or any other **anti-crawl limit** on it.
